File: WebUI/admin-app/src/app/models/workingHour.ts
export interface IWorkingHour {
  id: number;
  day: number;
  openTime: string;
  closeTime: string;
  hairSalonId: number;
}

export interface IWorkingHoursFormValues {
  hairSalonId: number;
  workingHours: IWorkingHour[];
}

File: Domain/Entities/WorkingHour.cs
using System;

namespace Domain.Entities
{
  public class WorkingHour
  {
    public int Id { get; set; }
    public int Day { get; set; }
    public TimeSpan OpenTime { get; set; }
    public TimeSpan CloseTime { get; set; }
    public int HairSalonId { get; set; }

    public virtual HairSalon HairSalon { get; set; }
  }
}